manifests/allow_from.pp

```diff
@@ -115,8 +115,14 @@
 
   ### SSHD_CONFIG
   # Defaults
-  $config_defaults = {
-    'notify' => Service[ sshd ],
+  if ( $sshd::manage_service ) {
+    $config_defaults = {
+      'notify' => Service[$sshd::service_name],
+    }
+  } else {
+    $config_defaults = {
+      'ensure' => 'present',
+    }
```

Comment:
I don't quite understand why we need "'ensure' => 'present'" only if we're not managing the service.

Reply:
Ah, I understand based on your comment in init.pp!